s3_file_field/_multipart.py

```diff
@@ -61,8 +59,16 @@ class MultipartManager:
 
     part_size = mb(64)
 
-    def initialize_upload(self, object_key: str, file_size: int) -> PresignedTransfer:
-        upload_id = self._create_upload_id(object_key)
+    def initialize_upload(
+        self,
+        object_key: str,
+        file_size: int,
+        content_type: Union[str, None] = None,
```

Comment:
```suggestion
        content_type: Optional[str] = None,
```

This is the more canonical syntax.